exercises/F-logical-operators/exercise2.js

```diff
@@ -5,7 +5,33 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(num) {
+  if (num < 0){
+    return true;
+  }
+  return false;
+}
+
+function isBetween5and10(num){
+  if(num >= 5 && num <=10){
+    return true;
+  }
+  return false;
+}
+
+function isShortName(personName){
+  if(personName.length < 7 && typeof personName === "string" ){
```

Comment:
Hi Hozan, everything looks great. For the whole exercise, you may want to consider doing away with the if conditions and return the conditions themselves.
 typeof personName === "string"  . Nice one here, it didn't occur to me to test for "stringness" 
Here is what I mean 
return name.length < 7; because this returns a boolean. 
 It may look unreadable now but I have seen it used in code examples and repos.

Reply:
Yes, for this exercise we'd like you to use the logical operators to return an answer from your function without using the `if... else...` blocks. You'll notice that you'll be able to write your function in 1 or 2 lines.

Try removing your `if...else...` blocks and returning the result of your expressions directly. Let me know if you have any questions.

Reply:
Hi Benyam, You are right, thanks for this suggestion, I will update this file. :)